crates/routes/src/utils/scheduled_tasks.rs

```diff
@@ -363,6 +370,59 @@ async fn active_counts(pool: &mut DbPool<'_>) -> LemmyResult<()> {
   Ok(())
 }
 
+async fn local_site_stats(pool: &mut DbPool<'_>) -> LemmyResult<()> {
+  info!("Updating local site stats ...");
+
+  let mut conn = get_conn(pool).await?;
+
+  let user_count: i64 = local_user::table
+    .inner_join(person::table.left_join(instance_actions::table))
+    // only count approved users
+    .filter(local_user::accepted_application)
+    // ignore banned and deleted accounts
+    .filter(instance_actions::received_ban_at.is_null())
+    .filter(not(person::deleted))
+    .select(count(local_user::id))
+    .get_result(&mut conn)
```

Comment:
If you look at these filters, they're actually telling us what triggers need to be added / fixed.

IE 

- on update of `instance_actions.received_ban`
- on update of `person.deleted`
- on update of `local_user.accepted_application`